README.md

```diff
@@ -138,6 +140,36 @@ function debug(plugins) {
 }
 ```
 
+##### `encapsulateStyles`
+
+* Type: `Boolean`
+* Default: `false`
+
+Resets CSS properties to their default values to effectively allow a component
+to opt out of CSS inheritance and be encapsulated from the rest of the
+application. There are two types of CSS properties that affect components, inherited (e.g. `font-size,`
```

Comment:
maybe we can mention that we are trying to reproduce the ShadowDOM behaviour by saying:
<blockquote>from the rest of the application similarly to ShadowDOM.</blockquote>
and clarify that this feature works only for Suit selectors so `.Component span {}` is not reset.